QUEST.tsv

```diff
@@ -884,9 +884,9 @@ QUEST_20150317_000883	$But the future I saw in that land had only death and regr
 QUEST_20150317_000884	$It is now up to the humans to fulfill that prophecy. {nl}The end lies beyond the fog.
 QUEST_20150317_000885	$I plan to leave the end of this story in the next revelation. {nl}Find my revelation in the Mage Tower.
 QUEST_20150317_000886	Laima
-QUEST_20150317_000887	There are still more keys left to find. {nl}It would be a relief if they are secretly guarded by Maven's plan.
+QUEST_20150317_000887	$There are still more Keys left to find. {nl}It's a relief that they are all secretly guarded by Maven's plan.
```

Comment:
You meant the keys right? I was wavering about whether to do it since I did it for Relics. Alrighty then. I'm pretty sure most folks will understand the keys are important